From 2dd000d6c2bfc862fe3aec48e740432226e235fe Mon Sep 17 00:00:00 2001
From: Marc Karasek <mkarasek@inspiresemi.com>
Date: Wed, 18 Sep 2024 13:29:43 -0500
Subject: [PATCH 19/21] scripts to build with RVI toolchain

This will config and build for the latest rvi build
under /opt/riscv64_080924
---
 buildrvi.sh | 1 +
 cleanrvi.sh | 1 +
 menurvi.sh  | 1 +
 3 files changed, 3 insertions(+)
 create mode 100755 buildrvi.sh
 create mode 100755 cleanrvi.sh
 create mode 100755 menurvi.sh

diff --git a/buildrvi.sh b/buildrvi.sh
new file mode 100755
index 000000000000..861ae385bc0e
--- /dev/null
+++ b/buildrvi.sh
@@ -0,0 +1 @@
+make ARCH=riscv CROSS_COMPILE=riscv64-unknown-linux-gnu- O=../linux_build DEBUG=1 -j16
diff --git a/cleanrvi.sh b/cleanrvi.sh
new file mode 100755
index 000000000000..8393ae5e4852
--- /dev/null
+++ b/cleanrvi.sh
@@ -0,0 +1 @@
+make ARCH=riscv CROSS_COMPILE=riscv64-unknown-linux-gnu- O=../linux_build DEBUG=1 clean
diff --git a/menurvi.sh b/menurvi.sh
new file mode 100755
index 000000000000..377192869e7a
--- /dev/null
+++ b/menurvi.sh
@@ -0,0 +1 @@
+make ARCH=riscv CROSS_COMPILE=riscv64-unknown-linux-gnu- O=../linux_build DEBUG=1 menuconfig
-- 
2.39.3

